pynng/nng.py

```diff
@@ -492,6 +492,28 @@ def remove_post_pipe_remove_cb(self, callback):
         """
         self._on_post_pipe_remove.remove(callback)
 
+    def recvmsg(self, block=True):
+        """
+        Return a Message object.
+
+        """
+        flags = 0
+        if not block:
+            flags |= lib.NNG_FLAG_NONBLOCK
+        msg_p = ffi.new('nng_msg **')
+        check_err(lib.nng_recvmsg(self.socket, msg_p, flags))
+        msg = msg_p[0]
+        lib_pipe = lib.nng_msg_get_pipe(msg)
+        pipe_id = lib.nng_pipe_id(lib_pipe)
+        if pipe_id < 0:
+            # TODO: Better exception
+            raise Exception('No such pipe')
```

Comment:
How about `PipeNotFoundError` much list `ModuleNotFoundError`?

Reply:
First I want to check to see what nng itself does whenever it's given a pipe object that doesn't exist anymore, and see if it makes sense to reuse that exception; otherwise, I like this.